src/Console/ElasticMigrateCommand.php

```diff
@@ -40,6 +41,16 @@ protected function getArguments()
         return $arguments;
     }
 
+    /**
+     * @return array
+     */
+    protected function getOptions()
+    {
```

Comment:
I believe there is a `parent:: getOptions` call missing, what do you think?

Reply:
I don't think the parent::getOptions is required. It's returning an empty array. But I'm happy to add it and use array_merge to merge the parent's in.

Reply:
The problem here, is that `$options` variable isn't defined before adding value in it.
So it's better to call `$options = parent::getOptions()` to be safe.